### PR TITLE
Home

### PR DESCRIPTION
**Pull request by Tass-Jomo**
image

### CODE REVIEW

**Comment by okothkongo** [comment]
Work on merge conflict 
How the about  the putting  navbar horizontally

File: config/dev.exs [hunk 1]
```diff
@@ -71,5 +71,5 @@ config :nailinda, Nailinda.Repo,
   username: "postgres",
   password: "postgres",
   database: "nailinda_dev",
-  hostname: "localhost",
+  hostname: "db",
```
Comment (sigu) [inline config/dev.exs]: Not part of this pull request

File: assets/css/app.css [hunk 2]
```diff
@@ -1,7 +1,16 @@
...
+@import "bulma";
 
  #map {
         height: 300px;
-      }
\ No newline at end of file
+
+      }
+
+.navbar-start .navbar-item {
```
Comment (sigu) [inline assets/css/app.css]: We can have the nav without any css styling

File: lib/nailinda_web/templates/layout/app.html.eex [hunk 3]
```diff
@@ -4,31 +4,56 @@
     <meta charset="utf-8"/>
     <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
-    <title>Nailinda | </title>
+    <title>Phauxth authentication with Phoenix</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bulma/0.7.4/css/bulma.min.css">
```
Comment (sigu) [inline lib/nailinda_web/templates/layout/app.html.eex]: We dont need to use cdn if we have bulma through `npm`

Comment (sigu) [reply lib/nailinda_web/templates/layout/app.html.eex]: Remove this link tag and make sure that the npm version is working correctly

**Review by sigu — CHANGES_REQUESTED** [review]
Generally okay, just make sure you use the npm version of bulma and remove the CDN one.

File: assets/css/app.scss [hunk 4]
```diff
@@ -0,0 +1,15 @@
+/* This file is for your main application css. */
+
+@import "bulma";
+
+ #map {
+        height: 300px;
+
+      }
+
+.navbar-start .navbar-item {
```
Comment (sigu) [inline assets/css/app.scss]: Not necessary

**Review by sigu — APPROVED** [review]
